src/components/network-map-tab.js

```diff
@@ -497,13 +491,12 @@ export const NetworkMapTab = ({
 
         Promise.all([substationPositionsDone, linePositionsDone])
             .then(() => {
-                setWaitingFullLoadGeoData(false);
+                setWaitingLoadData(false);
```

Comment:
To avoid duplication of this line, we could move it in a `Promise.finally` ?

Reply:
Yes totally, I do it